src/components/StyledText/StyledTitle/styled.ts

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components'
+import { FontSizes } from '../../../themes'
+
+type Props = {
+	changeColor?: string
+}
+
+export const TitleH1 = styled.h1<Props>`
+	font-size: ${FontSizes.SizeTitleH1};
+	text-align: center;
+	text-transform: uppercase;
+	color: ${({ changeColor }) =>
+		changeColor === 'darkBlue'
+			? ({ theme }) => theme.text.link // Dark blue color
```

Comment:
This comment seem unnecessary to me.

- These properties is dynamic.
- If we add new thema or changed old, we must edit this comments too.
- Comments should not hamper the sustainability of the code